### DISCUSSION TITLE
Video Meeting

### DISCUSSION THREAD

**Discussion by vgtom** [zitadel/zitadel#2242] [General]
Hi Team...can we have another video meeting like last time? I had some more clarifications!!

**Comment by adlerhurst** [comment]
Hi @vgtom

Thanks for reaching out.  
We are planning an open ask us anything session where you can join us in a meet. We would love to answer your questions there.

Cheers

**Reply by hifabienne** [reply]
Hi @vgtom 

We just announced our next ask us anything:
https://github.com/caos/zitadel/discussions/2272
https://www.linkedin.com/events/askusanything-september20216836585409703763968/

The event will take place on the 2. September
It would be great to meet you there.
